RHIDP-2895: Separate K8s module from OCP assembly for Helm CLI install (#325)
redhat-developer/red-hat-developers-documentation-rhdh: assemblies/assembly-rhdh-integration-aks.adoc

> [id='assembly-rhdh-integration-aks_{context}']

Reviewer: ```suggestion
[id="assembly-rhdh-integration-aks_{context}"]
```

> = {product} integration with {aks-brand-name} ({aks-short})

Reviewer: ```suggestion
= {product} integration with {aks-brand-name} ({aks-short})
:context:  assembly-rhdh-integration-aks

```
Try adding the context to the assemblies and see if it fixes your build error

include::modules/admin/proc-rhdh-deploy-aks-using-helm.adoc[leveloffset=+2]

> include::../artifacts/snip-install-helm-cli-k8s.adoc[leveloffset=+3]

Reviewer 2: ```suggestion
include::modules/installation/proc-installing-rhdh-on-kubernetes-with-helm-cli.adoc[leveloffset=+3]
```

Author: Doing it this way caused a build error, which is why the snippet was created
